fdbclient/S3BlobStore.actor.cpp

```diff
@@ -1186,12 +1186,13 @@ ACTOR Future<Reference<HTTP::IncomingResponse>> doRequest_impl(Reference<S3BlobS
 			}
 
 			Reference<HTTP::IncomingResponse> _r = wait(timeoutError(reqF, requestTimeout));
-			if (g_network->isSimulated() && deterministicRandom()->random01() < 0.1) {
+			if (g_network->isSimulated() && BUGGIFY && deterministicRandom()->random01() < 0.1) {
```

Comment:
Why do we do Buggify here? Thanks!

Reply:
Only do the random if BUGGIFY is enabled. I don't to turn off any variance for the moment in s3 client behaviors till we have stable determinism.